release/ray_release/glue.py

```diff
@@ -49,7 +48,7 @@
 command_runner_to_cluster_manager = {
     SDKRunner: FullClusterManager,
     ClientRunner: FullClusterManager,
-    JobRunner: MinimalClusterManager,
+    JobRunner: FullClusterManager,
```

Comment:
Probably also add `JobRunner: JobFileManager` to `command_runner_to_file_manager`